### PR TITLE
Fix meeting history using wrong user repo

### LINKED ISSUE #2885: All organization users displayed in participant history select
After accessing the account list on orga level all users are displayed in history participant select input. 
Only meeting users should be displayed there.

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #2885

### CODE REVIEW

**Review by luisa-beerboom — COMMENTED** [review]
Does this actually work if you select something?

**Review by emanuelschuetze — DISMISSED** [review]
Does not work for me. All accounts from other meetings are still listed in history select input - if you open account list before.

File: client/src/app/site/pages/meetings/pages/history/components/history-list/history-list.component.ts [hunk 1]
```diff
@@ -87,13 +87,13 @@ export class HistoryListComponent extends BaseMeetingComponent implements OnInit
             historyRepos.push(this.userRepo);
         }
         return historyRepos.mapToObject(repo => {
-            return { [repo.collection]: repo };
+            return { [repo.collection || repo.repo?.collection]: repo };
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/pages/history/components/history-list/history-list.component.ts:90]: You could add a  `collection` getter to the base-controller if you wanted

**Review by luisa-beerboom — APPROVED** [review]
Just one small optional remark

**Review by Elblinator — APPROVED** [review]
function

**Pull request merged**